exonum/src/node/mod.rs

```diff
@@ -676,7 +681,9 @@ impl NodeHandler {
     /// Returns start time of the requested round.
     pub fn round_start_time(&self, round: Round) -> SystemTime {
         let previous_round: u64 = round.previous().into();
-        let ms = previous_round * self.round_timeout();
+        let ms = previous_round * self.first_round_timeout()
+            + (previous_round * previous_round.saturating_sub(1)) / 2
```

Comment:
Could you please clarify this formulae?

Reply:
It is the sum of the arithmetic progression. For round r cumulative increase in round start time due to increasing round timeout will be (r-1)(r-2)/2 * round_timeout_increase. I'll clarify it in the comment.